caddyhttp/push/link_parser.go

```diff
@@ -0,0 +1,70 @@
+package push
+
+import (
+	"regexp"
+	"strings"
+)
+
+const (
+	commaSeparator     = ","
+	semicolonSeparator = ";"
+	equalSeparator     = "="
+)
+
+var (
+	resourceRegexp = regexp.MustCompile("^<(.*?)>;?(.*)$")
```

Comment:
It's a valid question, I'll dig into: https://tools.ietf.org/html/rfc5988#section-5.1